search-sentiments-map/app.js

```diff
@@ -14,19 +14,31 @@
 
 const express = require('express');
 const app = express();
-// Listen to the App Engine-specified port, or 3000 otherwise.
-const PORT = process.env.PORT || 3000;
+// Listen to the App Engine-specified port, or 4503 otherwise.
+const PORT = process.env.PORT || 4503;
 app.listen(PORT, () => {
   console.log(`Server listening on port ${PORT}...`);
 });
 
-// Use express to create server that displays the webpage with the html, css, and javascript
-// files in the public folder.
-app.use(express.static(__dirname +'/public'));
+// Use express to create server that displays the webpage with the html, css, 
+// and javascript files in the public folder.
+app.use(express.static('./public'));
 app.get('/', (req, res) => {
   res.sendFile('/index.html');
 });
 
-// Include the trends module so that it can be fetched from the client-side scripts.
 const trends = require('./routes/trends.js');
-app.use('/trends', trends);
+// Use the trends router so that it can be fetched from the client-side scripts.
+app.use('/trends', trends.router);
+
+// Uncomment the following line to get trends if none is in the datastore.
+// trends.updateTrendsFunction();
+
+var schedule = require('node-schedule');
+// Update top trends at minute 0 past every 12th hour (11am and 23pm every day).
+var j = schedule.scheduleJob('0 11,23 * * * *', function(){
```

Comment:
Swap to let not var.

Reply:
This is not a block scope but in a global scope, so I'd say use var instead of let?

Reply:
Ah okay